crescent/commands/groups.py

```diff
@@ -20,25 +20,50 @@
 @define
 class Group:
     name: str
+    description: Optional[str] = None
+    hooks: Optional[List[HookCallbackT]] = None
 
-    def sub_group(self, name: str) -> SubGroup:
-        return SubGroup(name, self.name)
+    def sub_group(
+        self,
+        name: str,
+        description: Optional[str] = None,
+        hooks: Optional[List[HookCallbackT]] = None,
+    ) -> SubGroup:
+        return SubGroup(
+            name=name,
+            parent=self,
+            description=description,
+            hooks=hooks,
```

Comment:
Should this be `hooks+self.hooks`, so that hooks on a Group are included in their sub groups?

Reply:
https://github.com/magpie-dev/hikari-crescent/pull/24/files#diff-638fdc4aef827417e5cee418cde5d206608d5c74d2ac7ca401400d31241b7201R67
That is covered here

Reply:
Ah ok, looks good then.